### DISCUSSION TITLE
Impossible to apply for GitHub Education Pack

### DISCUSSION THREAD

**Discussion by LyubomirT** [community/community#53199] [GitHub Education] [answered]
First, I have not been approved, since I had to use my device camera, that's understandable. I also got an error saying 

"You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?". 

Then, I attempted to use my device camera, and received the same error with the profile. I changed my name to how it appears in the image, I also placed a short bio, added a location, displayed the current time and the timezone, and added a status (removed that, after many fails). 

I'm not sure if it's a glitch on the platform, or am I just missing something? Any help appreciated.

**Comment by RexLee0929** [comment]
I also encountered the same problem, I tried many times but could not verify

**Reply by LyubomirT** [reply]
Did it give the same error to you?

**Reply by JoeTomkinson** [reply]
Had the exact same issues, all of tonight, still not able to verify, no clue what else I can reasonably do to get verified. i dont know how they expect us to falsify university email verification and camera photos of our IDs.

**Reply by LyubomirT** [reply]
I think that the problem is somewhere in the profile modification. They want us to place our real name + a short bio, but they don't tell us what name exactly to put.

**Comment by samus-aran** [comment]
Hi y'all 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 

Hope this helps! Let us know how you get along 👍

**Reply by LyubomirT** [reply]
Thanks for the information! I'll make sure to set everything as needed. Will this also require to change my username to the name written on the evidence? I'm also not quite sure how exactly to write my real name, since it's written with cyrillic letters on the document.

**Comment by LyubomirT** [comment, marked as answer]
# Solution


Due to my name being written with Cyrillic letters on the document, I had to get a translation of it and take a photo of it along with the original one. Then, I simply set the username on my profile exactly as it appears on the **translated** document, and after that I can try to verify. It may take a few tries if you have a bad camera, but eventually you'll succeed.

```
Credit: Maya from GitHub support
```

**Comment by mdtanvirahamedshanto** [comment]
You have a student id card can apply

**Reply by LyubomirT** [reply]
The problem is that my school does **NOT** provide any student ID cards, that's why I had to use my enrollment documents instead.

**Reply by mdtanvirahamedshanto** [reply]
You have any School Webmail
